### DISCUSSION TITLE
iOS Equivalent?

### DISCUSSION THREAD

**Discussion by mattkoch614** [skydoves/Pokedex#50] [Q&A]
This is a great project, and very helpful for showcasing a solid architecture choice for Android developers. Does anyone know if there is a similar/equivalent project for iOS, perhaps using something like Combine and SwiftUI?

Perhaps: https://github.com/nalexn/clean-architecture-swiftui/tree/mvvm ?

**Comment by amirisback** [comment]
I left my mark here, because I also want to know
